### ISSUE TITLE
Update used SDL2 version for the Windows build

### ISSUE THREAD

**Issue by segevfiner** [libretro/RetroArch#16717]
## Description

It seems like RetroArch for Windows is built with a rather old SDL2 version, this version has an annoying "feature" where it turns on the Switch Pro Controller LED. It would be nice if it is updated so it will stop doing that.

### Expected behavior

For RetroArch to ship with a new enough SDL2 that doesn't turn on the switch pro controller home led.

### Actual behavior

It doesn't. The home LED gets turned on when RetroArch is set to use the SDL2 controller driver.

### Steps to reproduce the bug

1. Open RetroArch with the SDL2 controller driver and connect a switch pro controller.

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.19.1

### Environment information

- OS: Windows 11 x64
- Compiler: [In case you are running local builds]

**Comment by segevfiner** [comment]
Created this as a workaround for now: https://gist.github.com/segevfiner/27558d5ba6ff2afeba07da656d531122

**Comment by gouchi** [comment]
@segevfiner for now, the workaround is to update `SDL2.dll`  using [2.32.10](https://github.com/libsdl-org/SDL/releases/tag/release-2.32.10).

[Source](https://github.com/libretro/RetroArch/issues/15196#issuecomment-4173074352)

**Comment by bcat** [comment]
+1 to this, there's a lot of nice fixes and improvements for NSO, 8BitDo, etc. controllers in more recent SDL versions. I've been using an updated DLL for some time now with no apparent issues.

**Comment by segevfiner** [comment]
Yeah... I would have even contributed an update if needed, but I'm not sure where is SDL2 picked up from in the Windows release build (It wasn't immediately obvious by just searching the repo) and didn't have the time to dig in further.

**Comment by gouchi** [comment]
We are using this [image](https://github.com/libretro/RetroArch/blob/1bb39379416c4b2900c2ab5b4d58d9bda100922c/.gitlab-ci.yml#L10) to build Windows and it seems we are using this [commit](https://git.libretro.com/libretro-infrastructure/libretro-build-mxe-win64-cross/-/blob/b155199a9fab6cbbf962ff4bb22f86876c299583/Dockerfile.gcc10#L11) for [mxe](https://mxe.cc/)  which will give us SDL2 [version 2.0.14](https://github.com/mxe/mxe/blob/ad07d47c3e43ec5f8d0b4cc4e5efa4e9a9a15426/src/sdl2.mk#L7).
